workdir/test-list.sh

```diff
@@ -0,0 +1,19 @@
+# List of regex that will match the tests you want to run
```

Comment:
Mention here that the regex match happens against the build scripts for all variables.